sdrbase/util/aprs.h

```diff
@@ -461,6 +461,7 @@ struct SDRBASE_API APRSPacket {
     int charToInt(QString &s, int idx);
     bool parseTime(QString& info, int& idx);
     bool parseTimeMDHM(QString& info, int& idx);
+    bool inRange(unsigned low, unsigned high, unsigned x);
```

Comment:
Obviously this is a very simple method just there to avoid writing the same condition over and over. In this case the body could be put here as an inline method.

Reply:
Moved to inline at the top of the file. Oops need to remove this from the `.h`